zebrad/tests/acceptance.rs

```diff
@@ -887,10 +887,22 @@ fn sync_until(
     if enable_mempool_at_height.is_some() {
         child.expect_stdout_line_matches("enabling mempool for debugging")?;
         child.expect_stdout_line_matches("activating mempool")?;
+
+        // make sure zebra is running with the mempool
+        child.expect_stdout_line_matches("verified checkpoint range block_count=400")?;
```

Comment:
This is testing that the zebrad is running with the mempool at the same time.

Reply:
Test Reliability: Is the exact block count important to this test?

Sometimes Zebra will commit two checkpoint ranges at the same time, so the block count would be 800.

```suggestion
        child.expect_stdout_line_matches("verified checkpoint range")?;
```

Reply:
Yea, that should be good enough. Done as part of https://github.com/ZcashFoundation/zebra/pull/2880/commits/ca88682b0e7e18092086cbfd39689bffb83c4a7a